### PR TITLE
Added support Debian jessie

### PR DESCRIPTION
**Pull request by grengojbo**

### CODE REVIEW

File: tools/st2ctl [hunk 1]
```diff
@@ -24,6 +24,9 @@ DEBTEST=`lsb_release -a 2> /dev/null | grep Distributor | awk '{print $3}'`
 if [[ "$DEBTEST" == "Ubuntu" ]]; then
   TYPE="debs"
   PYTHONPACK="/usr/lib/python2.7/dist-packages"
+elif [[ "$DEBTEST" == "Debia" ]]; then
```
Comment (DoriftoShoes) [inline tools/st2ctl]: Should this be "Debian" instead of "Debia"?

**Comment by sammcj** [comment]
```
root@dev-samm:~  # lsb_release -a 2> /dev/null | grep Distributor | awk '{print $3}'
Debian
```

**Comment by Kami** [comment]
@sammcj tried to run the modified script, but there are still some blockers - it fails when installing winexe package.

For now, we could skip installation of winexe package on Debian, but there might be other issues left. I will wait for @sammcj before merging this PR.

It's also worth noting that we only test it on Ubuntu and RHEL / Fedora right now so we can't guarantee everything will always work on Debian.

**Comment by sammcj** [comment]
```
root@dev-samm:~  # apt-get install winexe
Reading package lists... Done
Building dependency tree
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 winexe : Depends: libc6 (>= 2.15) but 2.13-38+deb7u8 is to be installed
E: Unable to correct problems, you have held broken packages.
root@dev-samm:~ 100 # apt-cache policy libc6
libc6:
  Installed: 2.13-38+deb7u8
  Candidate: 2.13-38+deb7u8
  Version table:
 *** 2.13-38+deb7u8 0
        500 http://apt/security.debian.org/ wheezy/updates/main amd64 Packages
        100 /var/lib/dpkg/status
     2.13-38+deb7u6 0
        500 http://apt/mirror.overthewire.com.au/debian/ wheezy/main amd64 Packages
root@dev-samm:~  # vi lol.sh
root@dev-samm:~  # INSTALL_WINDOWS_RUNNER_DEPENDENCIES=0 ./lol.sh
```



Installing without the winexe package downloads a lot of packages then fails with:

```
Installed /opt/openstack/mistral
Processing dependencies for mistral==0.0.5.dev694
Traceback (most recent call last):
  File "setup.py", line 30, in <module>
    pbr=True)
  File "/usr/lib/python2.7/distutils/core.py", line 152, in setup
    dist.run_commands()
  File "/usr/lib/python2.7/distutils/dist.py", line 953, in run_commands
    self.run_command(cmd)
  File "/usr/lib/python2.7/distutils/dist.py", line 972, in run_command
    cmd_obj.run()
  File "/opt/openstack/mistral/.venv/local/lib/python2.7/site-packages/distribute-0.6.24-py2.7.egg/setuptools/command/develop.py", line 27, in run
    self.install_for_development()
  File "/opt/openstack/mistral/.venv/local/lib/python2.7/site-packages/distribute-0.6.24-py2.7.egg/setuptools/command/develop.py", line 105, in install_for_development
    self.process_distribution(None, self.dist, not self.no_deps)
  File "/opt/openstack/mistral/.venv/local/lib/python2.7/site-packages/distribute-0.6.24-py2.7.egg/setuptools/command/easy_install.py", line 673, in process_distribution
    [requirement], self.local_index, self.easy_install
  File "/opt/openstack/mistral/.venv/local/lib/python2.7/site-packages/distribute-0.6.24-py2.7.egg/pkg_resources.py", line 589, in resolve
    requirements.extend(dist.requires(req.extras)[::-1])
  File "/opt/openstack/mistral/.venv/local/lib/python2.7/site-packages/distribute-0.6.24-py2.7.egg/pkg_resources.py", line 2207, in requires
    dm = self._dep_map
  File "/opt/openstack/mistral/.venv/local/lib/python2.7/site-packages/distribute-0.6.24-py2.7.egg/pkg_resources.py", line 2201, in _dep_map
    dm.setdefault(extra,[]).extend(parse_requirements(reqs))
  File "/opt/openstack/mistral/.venv/local/lib/python2.7/site-packages/distribute-0.6.24-py2.7.egg/pkg_resources.py", line 2507, in parse_requirements
    line, p, specs = scan_list(VERSION,LINE_END,line,p,(1,2),"version spec")
  File "/opt/openstack/mistral/.venv/local/lib/python2.7/site-packages/distribute-0.6.24-py2.7.egg/pkg_resources.py", line 2475, in scan_list
    raise ValueError("Expected "+item_name+" in",line,"at",line[p:])
ValueError: ('Expected version spec in', 'funcsigs;python_version<"3.3"', 'at', ';python_version<"3.3"')
```

**Comment by Kami** [comment]
This looks like it could be an issue with using an older version of pip, although we install the latest version in the script, so not exactly sure what is going on.

**Comment by lakshmi-kannan** [comment]
Related: https://github.com/StackStorm/st2/pull/1774/files

**Comment by jfryman** [comment]
Thanks for the PR. We're re-addressing how we are doing installation, so this is no longer applicable. Will have official Jessie support very soon!

**Pull request closed without merging**